stack/python/stack.py

```diff
@@ -0,0 +1,38 @@
+class Stack:
+    def __init__(self):
+        self.stack = []
+
+    def push(self, dataval):
+        if dataval not in self.stack:
```

Comment:
Does it really matter if the dataval is already on the stack?

Reply:
Well, there is no guideline on what matters and what not. I will remove the line.

Reply:
Resolved